app/controllers/employers/census_employees_controller.rb

```diff
@@ -263,6 +263,14 @@ def benefit_group
     @census_employee.benefit_group_assignments.build unless @census_employee.benefit_group_assignments.present?
   end
 
+  def change_expected_selection
+    census_employees = CensusEmployee.find(params[:ids])
+    census_employees.each do |census_employee|
+      census_employee.update_attributes(:expected_selection=>params[:expected_selection])
+    end
```

Comment:
Add a flash notice saying "successfully submitted the selected Employees participation status"